src/in-house-liquidity/index.test.ts

```diff
@@ -54,4 +65,217 @@ describe('In House Liquidity Calls', () => {
       ).toThrow('Cannot call IHL because walletAddress is null')
     })
   })
+
+  describe('makeRequest', () => {
+    it('makes a FC authenticated call', async () => {
+      ;(getAuthHeaders as jest.Mock).mockResolvedValueOnce({ 'header-key': 'header-val' })
```

Comment:
you can used [jest.mocked(...)](https://jestjs.io/docs/mock-function-api/#jestmockedsource-options) instead of `as`. Or actually `mocked` helper from `ts-utils/jest` since we are in an older version of jest

Reply:
This introduces another issue with TS complaining about resolved result type :frowning_face: